src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -225,6 +225,7 @@ public void Write(BinaryAnalyzerContext context, CompilerData compilerData)
                 { "language", compilerData.Language },
                 { "debuggingFileName", compilerData.DebuggingFileName ?? string.Empty },
                 { "debuggingGuid", compilerData.DebuggingFileGuid ?? string.Empty },
+                { "debuggingFileLastModifiedDateUTC", compilerData.DebuggingFileLastModifiedDateUtc },
```

Comment:
>compilerData.DebuggingFileLastModifiedDateUtc [](http://example.com/codeflow?start=54&length=45)

the syntax here is different than the other strings
`compilerData.DebuggingFileGuid ?? string.Empty`
because the default of string is null not string empty
I think we should have this as well?

Reply:
Makes sense/ Fixed.